hack/update-codecgen.sh

```diff
@@ -54,7 +54,7 @@ function depends {
   file=${generated_files[$1]//\.generated\.go/.go}
   deps=$(go list -f "{{.Deps}}" ${file} | tr "[" " " | tr "]" " ")
   inputfile=${generated_files[$2]//\.generated\.go/.go}
-  fullpath=$(readlink -f ${generated_files[$2]//\.generated\.go/.go})
+  fullpath=${generated_files[$2]//\.generated\.go/.go}
```

Comment:
oh, right, this is broken on OS X, but I should revert it here and do it in a different PR.